data-prepper-api/src/main/java/com/amazon/dataprepper/model/log/JacksonLog.java

```diff
@@ -17,6 +18,7 @@
 
     protected JacksonLog(final Builder builder) {
         super(builder);
+        this.put("event_timestamp", LocalDateTime.now());
```

Comment:
I think we may want to use `OffsetDateTime` rather than `LocalDateTime`. Using `OffsetDateTime` will include the time zone.